runtime/interpreter/value.go

```diff
@@ -14800,6 +14803,9 @@ func (v *CompositeValue) Clone(interpreter *Interpreter) Value {
 		panic(ExternalError{err})
 	}
 
+	elementMemoryUse := common.NewAtreeMapPreAllocatedElementsMemoryUsage(v.dictionary.Count())
+	common.UseMemory(interpreter.memoryGauge, elementMemoryUse)
+
```

Comment:
Just a note:  don't need to meter memory in `Clone()`.  We can remove this later.